src/ixdat/exporters/ms_exporter.py

```diff
@@ -0,0 +1,44 @@
+from . import CSVExporter, SpectrumSeriesExporter
+
+
+class MSExporter(CSVExporter):
+    """By default, everything from an MS Measurement is exported."""
+
+    pass
+
+
+class MSSpectroExporter(MSExporter):
+    def __init__(self, measurement, delim=","):
+        super().__init__(measurement, delim=delim)
+        # self.spectra_exporter = SpectrumSeriesExporter(measurement.spectrum_series)
+        # FIXME: Have to do a property because this __int__ gets called before the
+        #    measurement's __init__ is finished...
+        self._spectra_exporter = None
```

Comment:
Is it possible that the `SpectrumSeriesExporter` would take the whole `measurement` only like this exporter is doing? I sense an inconsistency